test/unit/test.speech_to_text.v1.js

```diff
@@ -344,4 +344,147 @@ describe('speech_to_text', function() {
       assert(isStream(speech_to_text.createRecognizeStream()));
     });
   });
+
+  describe('asynchronous callback api', function() {
+    it('should register new callback url', function() {
+      const path = '/v1/register_callback?callback_url=http%3A%2F%2Fwww.callback.fr%2Fresults&user_secret=ThisIsMySecret';
+      const response = {
+        status: 200,
+        url: 'http://www.callback.fr/results'
```

Comment:
I know this one is offline but please use an IBM-owned domain here as well - e.g. ibm.com or watson-test-resources.mybluemix.net